tests/test_flow.py

```diff
@@ -3225,43 +3230,9 @@ def test_csv_reader_parquet_write_microsecs(tmpdir):
     controller.await_termination()
     read_back_df = pd.read_parquet(out_file, columns=columns)
 
-    assert read_back_df.equals(expected), f"{read_back_df}\n!=\n{expected}"
-
-
-def test_csv_reader_parquet_write_nanosecs(tmpdir):
```

Comment:
@assaf758, are we ok with dropping support for this? This was actually requested a couple of years back.

Reply:
Ability to read CSV with nanosecond precision, iirc. I think it was related to work done by OrZ (like I said, a few years back). I don't remember any details beyond that.

Reply:
Will double check

Reply:
Since Python doesn't inherently support nanosecond resolution, the purpose of this test is to assess Storey's capacity to read values in nanosecond format from a CSV file and convert them to microseconds. I will reinstate the test and update its name to more accurately describe this capability.